core/src/main/scala/org/apache/spark/util/HadoopFSUtils.scala

```diff
@@ -187,10 +187,12 @@ private[spark] object HadoopFSUtils extends Logging {
 
   // scalastyle:off argcount
   /**
-   * Lists a single filesystem path recursively. If a `SparkContext` object is specified, this
-   * function may launch Spark jobs to parallelize listing based on `parallelismThreshold`.
+   * Lists a single filesystem path recursively.
+   *
+   * 1. If contextOpt is defined, this function may launch Spark jobs to parallelize listing
+   * based on `parallelismThreshold`.
    *
-   * If sessionOpt is None, this may be called on executors.
+   * 2. If contextOpt is None, this may be called on executors.
```

Comment:
I don't think this makes the comments particulaily easier to read. The only valid fix here is the parameter name which is super trivial.